README.md

```diff
@@ -442,10 +444,322 @@ If you want to add test data (collections and datasets) to the Dataverse instanc
+    cy.customMount(<Home datasetRepository={datasetRepository} />)
+    cy.findByRole('heading').should('contain.text', 'Root')
+  })
+})
+```
+
+</details>
+
+### 2. **Integration Tests:**
+
+Test the integration of the SPA with external systems, such as APIs, third-party
+libraries (like js-dataverse), or databases. This ensures that the application communicates correctly with outside
+resources and services.
+
+- **External Integrations:** Test the integration of the SPA with external systems, such as APIs, third-party
+  libraries (like [js-dataverse](https://github.com/IQSS/dataverse-client-javascript/pkgs/npm/dataverse-client-javascript/97068340)), or databases.
+- **Tools and Frameworks:** We use Cypress for integration tests, to test the repository's implementation.
+
+<details>
+<summary><strong>Example of an Integration Test</strong></summary>
+
+```javascript
+//tests/e2e-integration/integration/datasets/DatasetJSDataverseRepository.spec.ts
+
+describe('Dataset JSDataverse Repository', () => {
+  before(() => TestsUtils.setup())
+  beforeEach(() => {
+    TestsUtils.login()
+  })
+  it('gets the dataset by persistentId', async () => {
+    const datasetResponse = await DatasetHelper.create()
+
+    await datasetRepository.getByPersistentId(datasetResponse.persistentId).then((dataset) => {
+      if (!dataset) {
+        throw new Error('Dataset not found')
+      }
+      const datasetExpected = datasetData(dataset.persistentId, dataset.version.id)
+
+      expect(dataset.license).to.deep.equal(datasetExpected.license)
+      expect(dataset.metadataBlocks).to.deep.equal(datasetExpected.metadataBlocks)
+      expect(dataset.summaryFields).to.deep.equal(datasetExpected.summaryFields)
+      expect(dataset.version).to.deep.equal(datasetExpected.version)
+      expect(dataset.metadataBlocks[0].fields.publicationDate).not.to.exist
+      expect(dataset.metadataBlocks[0].fields.citationDate).not.to.exist
+      expect(dataset.permissions).to.deep.equal(datasetExpected.permissions)
+      expect(dataset.locks).to.deep.equal(datasetExpected.locks)
+      expect(dataset.downloadUrls).to.deep.equal(datasetExpected.downloadUrls)
+      expect(dataset.fileDownloadSizes).to.deep.equal(datasetExpected.fileDownloadSizes)
+    })
+  })
+})
+```
+
+</details>
+
+<details>
+<summary><strong>Wait for no locks</strong></summary>
+
+Some integration tests require waiting for no locks to be present in the dataset. This is done using the `waitForNoLocks`
+method from the `TestsUtils` class.
+
+```javascript
+it('waits for no locks', async () => {
+  const datasetResponse = await DatasetHelper.create()
+
+  await DatasetHelper.publish(datasetResponse.persistentId)
+  await TestsUtils.waitForNoLocks(datasetResponse.persistentId)
+
+  await datasetRepository.getByPersistentId(datasetResponse.persistentId).then((dataset) => {
+    if (!dataset) {
+      throw new Error('Dataset not found')
+    }
+    expect(dataset.locks).to.deep.equal([])
+  })
+})
+```
+
```

Comment:
Is there a missing `</details>` here? When I look at the page at https://github.com/IQSS/dataverse-frontend/blob/1220a7d865f3a0031e8fa8449265855f28e5200c/README.md it seems to end at "Wait for no locks":

![Screenshot 2024-03-21 at 3 10 23 PM](https://github.com/IQSS/dataverse-frontend/assets/21006/4574a167-5291-47b3-9c12-26e252c535fc)

If I expand it, I see more content, unrelated to locks. The rest of the page, basically.